tmva/sofie/inc/TMVA/ROperator_BasicUnary.hxx

```diff
@@ -0,0 +1,79 @@
+#ifndef TMVA_EXPERIMENTAL_SOFIE_ROPERATOR_BASIC_UNARY
+#define TMVA_EXPERIMENTAL_SOFIE_ROPERATOR_BASIC_UNARY
+
+#include <TMVA/ROperator.hxx>
+#include <TMVA/RModel.hxx>
+#include <TMVA/SOFIE_common.hxx>
+
+namespace TMVA {
+namespace Experimental {
+namespace SOFIE {
+
+enum class EBasicUnaryOperator { kReciprocal, kSqrt };
```

Comment:
Can we include the Negation operator here as well?

cc: @lmoneta

Reply:
I added the Exp operator and moved Neg to ROperator_BasicUnary as we have discussed.